src/utils/constants.ts

```diff
@@ -3,7 +3,8 @@ export const API_URL = 'https://clima.arpa.veneto.it/backend/'; //process.env.RE
 export const V2_API_URL = 'https://clima.arpa.veneto.it/backend/'; //process.env.REACT_APP_API_URL as string;
```

Comment:
I would clean this file up, as it is getting confusing. Which of these variables are actually used in the code, if any?